# init method rviz or gps
align_method: 1  # 0 ndt_autoware 1 ndt_omp

# imu /odom param
use_odom: false
use_imu: true
invert_imu: true
imu_upside_down: false

# points range
min_scan_range: 2
max_scan_range: 80
circle_radius: 100

# vox
scan_leaf: 0.5
map_leaf: 1.0

# ndt
res: 2.0
iter: 30   # ndt迭代次数限制
epsilon: 0.01
step: 0.1
scoreThreshold: 1

# map 其他地图LLA 114.044083877  22.6523610167   67.831
map_topic: /points_map
path: /home/xchu/vector_map_data/points_map/fushikang/map.pcd

# topic list
imu_topic: /imu/data
odom_topic: /golfcar/odom
gps_topic: /novatel718d/pos
heading_topic: /novatel718d/heading
lidar_topic: /top/rslidar_points

